Session.js

```diff
@@ -0,0 +1,38 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the client side and be used as a buffer  
+// for client to server information needed for other components of Virtual Movie Night.
+ 
+    constructor(sessionId) {
+        this.sessionId = sessionId;
+    }
+    constructor(instanceIp) {
+        this.instanceIp = instanceIp;
+    } 
+    constructor(listOfAttendees){
+        this.listOfAttendees = listOfAttendees;
+    }
+    
+    constructor(controllerIp){
+        this.controllerIp = controllerIp;
+    }
```

Comment:
this can be made into one constructor and the variables can be private. i don't really know what the constructor would take in so honestly you could probably just delay these classes until the datastore manager is finished.

check the style guides for how to make things private and denoting that in documentation

Reply:
Okay